### ISSUE TITLE
Key contacts form on institution page no longer works

### ISSUE THREAD

**Issue by xray7224** [OpenTechStrategies/swoptact#128]
I noticed when trying to tack #74 that the ability to add key contacts no longer works, the input boxes don't appear in the DOM. I searched back in the commit history to find out which commit did it, it seems it's https://github.com/OpenTechStrategies/swoptact/commit/27a61aa51f3eaa1807d5d89ed7719682ecb5d614 (@cecilia-donnelly).

I've marked this as high priority as #74 is and it depends on this being fixed.

**Comment by cecilia-donnelly** [comment]
@xray7224:
So, there are a couple things going on here.  The phone number never had an input box because we weren't correctly linking Participants to Contacts.  Adding that input box so phone numbers could be added and edited was the original purpose of issue #74.  Because of #121, the title ended up being in the same situation as the phone number.  It, too, is now being pulled from the Participant model instead of the Contact model, so the input box disappeared.  I edited #74 accordingly, to make clear that titles should be managed the same way as phone numbers.

Now, the name field is a totally different matter.  That disappeared in 
acb6c45, I think.  I can't easily check because of the db change since then (checking that revision out and trying to look at the Institution page gives an error), but I will check and confirm. 

In short, though, the missing title and phone number inputs are expected, and the missing name input box is not expected.  So, adding the title and phone number input boxes back to the form is _part_ of #74.

Does that help?

**Comment by xray7224** [comment]
It does, thanks for clarifying.

**Comment by cecilia-donnelly** [comment]
#94 changes have fixed this as of eba18a7.
